modelexpress_client/python/tests/test_artifact_health_url.py

```
@@ -0,0 +1,120 @@
+        al.resolve_health_url(NON_DEFAULT_HEALTH, VLLM_DEFAULT, "fd00::1")
+        == "http://[fd00::1]:9090/health"
+    )
+
+
+@pytest.mark.parametrize(
+    ("host", "expected"),
+    [
+        ("127.0.0.1", True),
+        ("127.1.2.3", True),
+        ("localhost", True),
+        ("0.0.0.0", True),
```

Ruff (0.15.21)

[error] Possible binding to all interfaces

(S104)